src/components/NewLocationPage/SummaryStatsBlock/MetricSubLabel.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * For daily new cases + percent vaccinated.
+ * "Per 100k" // "1+ dose"
+ */
+
+import React from 'react';
+import { SubLabel, SubLabelWrapper } from './SummaryStatsBlock.style';
+
+const MetricSubLabel: React.FC<{ text: string[]; isMobile: boolean }> = ({
+  text,
+  isMobile,
+}) => {
+  return (
+    <SubLabelWrapper>
+      <SubLabel>{text[0]}</SubLabel> {!isMobile && <br />}
+      <SubLabel>{text[1]}</SubLabel>
+    </SubLabelWrapper>
```

Comment:
How about making the text wrap and making the wrapper have a maximum fit instead of forcing the break? We can keep it as is if that's too much trouble!